core/jazz_cloud-logs-streamer/components/config.js

```diff
@@ -41,6 +41,8 @@ var getStageConfig = (event, context) => {
       stage = 'stg';
     } else if (fnName.endsWith('prod')) {
       stage = 'prod';
+    } else {
```

Comment:
Can you provide context as to what scenarios is this needed in? Introducing this will break environment management.

Reply:
When writing tests for the buildRequest and post function, end of the function name used is "-test". This test file contains the ES_ENDPOINT: "abc.us-east-1.es.amazonaws.com" which is suitable to run the tests for the post and buildRequest functions

Reply:
The tests should fall into this other `if` block at the top here: https://github.com/suryajak/jazz/pull/8/files#diff-4c6e3fdc46524adc9896feae482fa946R32.

Introducing an else block here opens up the code for a category of errors. For changes you are trying to introduce in the actual code always think of the impact area. One of the ways is to follow all the places where it is being used and see how that affect the callers.

Reply:
Ok surya, i understand what you are saying, I will remove this else statement, however, the test-configs.json file is still needed because without the buildrequest function does not have a config file to call when using the "-test" suffix

Reply:
Take a look at the link I posted above - it should help you understand how to construct an `event` where the config file is a test one.